cinch/roles/nginx/handlers/main.yml

```diff
@@ -0,0 +1,8 @@
+- name: reload systemd
```

Comment:
only in Ansible 2.4 and above. I guess we claim compatibility also with lower versions, mainly since 2.4 is not yet available even on Fedora...